modules/apps/segments/segments-api/src/main/java/com/liferay/segments/manager/SegmentsExperienceManager.java

```diff
@@ -47,8 +47,14 @@ public long getSegmentsExperienceId(HttpServletRequest httpServletRequest) {
 			(ThemeDisplay)httpServletRequest.getAttribute(
 				WebKeys.THEME_DISPLAY);
 
+		long plid = 0;
+
+		if (themeDisplay != null) {
```

Comment:
As we are waiting for Victor, solution, better if we keep the NullPointerException error, instead to serve a bad experience.

Reply:
@cgoncas, should we cherry-pick this: https://github.com/jkappler/liferay-portal/pull/1089

Reply:
Hey @jkappler !!

i don't think this fix the problem :(.

I have ask Victor more information in https://github.com/jkappler/liferay-portal/pull/1089#issuecomment-1319675903

Reply:
@cgoncas  I do not like it, but  ok I am going to delete the if.

Reply:
ci:test:relevant